.pre-commit-config.yaml

```diff
@@ -43,33 +43,29 @@ repos:
     hooks:
       - id: pylint
         args: [
-          "-rn", # Only display messages
-          "--rcfile=tests/pylintrc", # Link to your config file
-        ]
+            "-rn", # Only display messages
+            "--rcfile=tests/pylintrc", # Link to your config file
+          ]
```

Comment:
please revert the auto IDE formatting changes (unless you think they are needed)